projects/components/src/combo-box/combo-box.component.ts

```diff
@@ -79,6 +85,7 @@ import { ComboBoxMode, ComboBoxOption, ComboBoxResult } from './combo-box-api';
 
           <!-- Clear Button -->
           <div
+            *ngIf="!this.disabled"
```

Comment:
is this required? Mostly popover would block all interactions anyway right>?

Reply:
Yes it's required. The popover is not blocking the x but it should.